ml-agents-envs/mlagents/envs/brain.py

```diff
@@ -161,8 +162,9 @@ def process_pixels(image_bytes: bytes, gray_scale: bool) -> np.ndarray:
         :param image_bytes: input byte array corresponding to image
         :return: processed numpy array of observation from environment
         """
-        image_bytearray = bytearray(image_bytes)
-        image = Image.open(io.BytesIO(image_bytearray))
+        with hierarchical_timer("image_decompress"):
+            image_bytearray = bytearray(image_bytes)
+            image = Image.open(io.BytesIO(image_bytearray))
```

Comment:
Actually, I might need to add the next line under the timer too. According to the [docs](https://pillow.readthedocs.io/en/4.2.x/reference/Image.html#PIL.Image.open), `Image.open()` 
> is a lazy operation; this function identifies the file, but the file remains open and the actual image data is not read from the file until you try to process the data (or call the load() method).

Reply:
👍